lighthouse/transparent-multicluster-and-namespace-mapping-support.md

```diff
@@ -0,0 +1,84 @@
+# Transparent multicluster support
+
+## Summary
+
+Lighthouse is a component from Submariner which can be used independently of the dataplane components,
+and exchanges information in the form of ServiceImport and EndpointSlice CR/CRDs over the broker.
+
+Lighthouse implements a CoreDNS plugin to perform dns resolution for clusterset services.
+
+Lighthouse implements the
+[MCS API](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api)
+with some additional extensions.
+
+In terms of DNS resolution, the MCS API specifies that multicluster services can be resolved with the following
+FQDN format:
+
+* $service-name.$namespace.svc.clusterset.local
+
+In the context of [transparent multicluster](https://github.com/kcp-dev/kcp/tree/main/docs/investigations),
+two extra features are necessary:
+
+* The ability to resolve multicluster services via $service-name.$namespace or $service-name, in a way
```

Comment:
comment from @smarterclayton on slack: we would also need to resolve $service-name.$namespace.svc.cluster.local as the multicluster service